lib/src/services/jira_api.ts

```diff
@@ -570,48 +585,162 @@ const jiraClient = (
             ],
             expand: ["changelog"],
           }),
+
+        //   const fs = require("fs");
```

Comment:
?

Reply:
Section removed. Seems to be old debug code